sktime/forecasting/compose/_reduce.py

```diff
@@ -57,15 +58,17 @@ def update(self, y, X=None, update_params=False):
         self : an instance of self
         """
         if X is not None or update_params:
-            raise NotImplementedError()
+            raise NotImplementedError(
+                "Exogenous variables `X` and `update_params` are not yet supported."
+            )
```

Comment:
Once we have merged #660, we need to remove the `or update_params` here and update the error message.

Reply:
Now merged